nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml

```diff
@@ -19,6 +19,7 @@
           skip-tag: true
           wipe-workspace: true
           basedir: "nfs-ganesha-debian"
+
```

Comment:
i couldn't find a way to do it in the GUI the same way i can change around the build and setup scripts. If the rest of it is fine, we can just merge it and hope for the best?